mod/ckeditor/elgg-plugin.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+	'home_language_keys' => [
+		'ckeditor:html',
+		'ckeditor:visual',
+	],
+];
```

Comment:
The only purpose of the `elgg-plugin.php` file is to define the most commonly used translation strings?